src/components/map/Map.tsx

```diff
@@ -25,6 +25,18 @@ type Props = {
 
 const MARKER_ICON_SIZE = normalize(40);
 
+const MapIcon = ({
+  iconColor,
+  iconName = 'location'
+}: {
+  iconColor?: string;
+  iconName?: string;
+}) => {
+  const MarkerIcon = Icon[_upperFirst(iconName) as keyof typeof Icon];
+
+  return <MarkerIcon color={iconColor} size={MARKER_ICON_SIZE} />;
+};
+
```

Comment:
If the category does not have an icon, i.e. `iconName === null`, such an error occurs. Can we add an extra control here?


<img width="289" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/b44ce87c-870b-4fcc-962b-b3c812030638">
<img width="299" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/e2b8777a-bd12-4df1-8f44-a5c922246284">

Reply:
From which screen or component were you navigating? `'location'`. Where did you place the `console.log`? `iconName` should be string or undefined, to fall back to

Reply:
route = POI -> Bildung -> Kartenansicht 

I placed the log in the `MapIcon`.
<img width="439" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/11755668/85fd5199-68d1-44cb-a9e2-1b9d5ef160b1">

Reply:
I updated it to `iconName={marker.iconName ? marker.iconName : undefined}`